opennms-services/src/test/java/org/opennms/netmgt/nb/Nms0001NetworkBuilder.java

```diff
@@ -52,10 +52,10 @@
     FROH_IP_IF_MAP.put(InetAddressUtils.addr("10.0.1.2"), 606);
     FROH_IP_IF_MAP.put(InetAddressUtils.addr("66.3.9.1"), 613);
     FROH_IP_IF_MAP.put(InetAddressUtils.addr("10.246.16.245"), 608);
-    FROH_IP_IF_MAP.put(InetAddressUtils.addr("66-3-123-1.la-04.cvx.algx.net"), 668);
```

Comment:
Is it just that these IPs are no longer accessible and throwing exception in static block that would make all other elements to be not added ?

Reply:
Yeah, they're not IPs at all, and don't parse.  It honestly makes no sense to me why this wasn't failing for all of time. The _entire_ block of map-filling doesn't happen.